docs/technical-changelog.mdx

```diff
@@ -381,138 +381,138 @@ This page documents all notable changes to Sourcegraph. For more detailed change
 - Workaround settingscasacde error checks `(PR #3364)`
 
 #### Smart-Apply
 
 - Change the deployment id for the custom model `(PR #3921)`
-  - fix(smart-apply): Change deployment id
+  - Fix(smart-apply): Change deployment id
 
 #### Source
 
-- sub-repo-perms: adjust ipv4 string prefix matcher to use stack instead of heap allocations `(PR #4054)`
+- Sub-repo-perms: adjust ipv4 string prefix matcher to use stack instead of heap allocations `(PR #4054)`
   - N/A
 
 #### Workspaces
 
-- only check eTLD+1 if domain might have a subdomain `(PR #3561)`
+- Only check eTLD+1 if domain might have a subdomain `(PR #3561)`
 
 #### Workspaces/Web
 
-- fix tag manager ID interpolation `(PR #3664)`
+- Fix tag manager ID interpolation `(PR #3664)`
 
 #### Others
 
-- escape LIKE queries for users `(PR #4311)`
-- add ssh-keygen to gitserver to enable commit signing `(PR #4300)`
-- clean up indexes for user_relevant_repos `(PR #4281)`
-- skip check for github app commit signing if configured by SSH `(PR #4169)`
-- init subrepo perms before enterprise init hooks `(PR #4064)`
-- only show commit signing for gitlab and github `(PR #4027)`
-- unset cxx instead of NoCC `(PR #3958)`
-  - na
-- use pathspec literal to avoid colons `(PR #3928)`
+- Escape LIKE queries for users `(PR #4311)`
+- Add ssh-keygen to gitserver to enable commit signing `(PR #4300)`
+- Clean up indexes for user_relevant_repos `(PR #4281)`
+- Skip check for github app commit signing if configured by SSH `(PR #4169)`
+- Init subrepo perms before enterprise init hooks `(PR #4064)`
+- Only show commit signing for gitlab and github `(PR #4027)`
+- Unset cxx instead of NoCC `(PR #3958)`
+  - Na
+- Use pathspec literal to avoid colons `(PR #3928)`
   - Fixed a bug where paths that start with colons may be reported as not found.
 - (Agents) Modify agent submenu styling `(PR #3810)`
 - (Agents) Remove description field in agent creation step `(PR #3808)`
 - Enforce file size limits for highlighting `(PR #3697)`
   - Enforces limits of 1.5 MiB and 50K lines for highlighting code files.
-- remove vertical scrollbar from pre block `(PR #3678)`
+- Remove vertical scrollbar from pre block `(PR #3678)`
   - (minor bugfix) Fixed an issue where we would show scrollbars on unscrollable items in the dynamic search filters sidebar
-- fetch modified files for `DiffModifiesFile` `(PR #3654)`
-- clean up cmd leak `(PR #3642)`
-- a few bugfixes for the experimental commit search on searcher `(PR #3639)`
-- close diff reader `(PR #3636)`
-- a few small fixes for commit search on searcher experiment `(PR #3620)`
+- Fetch modified files for `DiffModifiesFile` `(PR #3654)`
+- Clean up cmd leak `(PR #3642)`
+- A few bugfixes for the experimental commit search on searcher `(PR #3639)`
+- Close diff reader `(PR #3636)`
+- A few small fixes for commit search on searcher experiment `(PR #3620)`
 - Nightly Minor Release Pipeline `(PR #3609)`
   - NA
 - /latest releaseregistry endpoint returns latest by semver `(PR #3563)`
```

Comment:
Maybe this is a misspellling or something? seems like `releaseregistry` should be 1 word

Reply:
It's a service name, and we generally refer to it as one word for some reason 🤷 😆 

But really, it shouldn't be in there at all as it's an internal service 👀 
Looks like despite being `N/A` in the changelog for a few of these, the author forgot to add the `no-changelog` label so it was still included.